### PR TITLE
Fixing yAxisLabel in CGE chart

### LINKED ISSUE #384: Update CGE units
Image

### PR DESCRIPTION
**Pull request by Vismayak**

### CODE REVIEW

File: src/components/childComponents/outputWidgets/CGE.jsx [hunk 1]
```diff
@@ -259,6 +259,7 @@ export default function CGE(props) {
 												<CGEBarChart
 													region={region}
 													data={cgeHouseholdIncomeContent}
+													yAxisLegend="Million of $"
```
Comment (longshuicy) [inline src/components/childComponents/outputWidgets/CGE.jsx]: Household income is also needs to be Million of $
See line:
https://github.com/IN-CORE/incore-playbook/pull/391/files#diff-c5bfe74f970a4d1737b9a2dc8488c94312a2fff87b17338112062a75acb8fee6R267

Comment (Vismayak) [reply src/components/childComponents/outputWidgets/CGE.jsx]: image
It is showing as such no?

Comment (Vismayak) [reply src/components/childComponents/outputWidgets/CGE.jsx]: Also would it not make more sense for the label to be for the x axis?

**Comment by Vismayak** [comment]
I think the xAxislabel is now showing correctly

image

**Review by longshuicy — APPROVED** [review]
Looks great. Good job on fixing the x-axis label.

**Pull request merged**